### PR TITLE
Phrasing, grammar, and spellcheck changes in files starting with A-H

### PR DESCRIPTION
**Pull request by cjrkoa**
I didn't want to do all the files in one mega PR so I decided to break this up a little

### CODE REVIEW

File: balancer-design.typ [hunk 1]
```diff
@@ -7,7 +7,7 @@
...
+Shown in @Figure::balancer-internals-class is the internal structure of the Balancer. The Balancer will discover monoliths using the `MonolithDiscoverer` (a process further described in @Chapter::ServiceDiscovery), and the `MonolithConnectionManager` will establish connections to each monolith. `Balancer` will update `BalancerContext` according to the messages it receives from `BalancerLink`. `Balancer` will then use `BalancerContext` to route messages to the appropriate monoliths. Clients work similarly, except that they establish connections to the Balancer via `BalancerService`. `BalancerService` handles proxying HTTP requests to the appropriate monoliths, and also accepting and upgrading WebSocket connections.
```
Comment (dyc3) [inline balancer-design.typ]: Generally, we capitalize "Monolith" and "Balancer" so I think we should keep that.

File: .DS_Store
Comment (dyc3) [inline .DS_Store:1]: Don't commit this, add it to the gitignore

**Comment by cjrkoa** [comment]
Idk what happened with this branch or why the client reconnect sequence diagram is here

**Review by dyc3 — CHANGES_REQUESTED** [review]
Most of these aren't typos, they are grammar and paraphrasing changes.

**Comment by cjrkoa** [comment]
Something is wrong with this entire branch, I'm just gonna close this and start fresh

**Pull request closed without merging**